docs/lab/aws/ooto-driver.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: kmm.sigs.k8s.io/v1beta1
+kind: Module
+metadata:
+  name: ooto-ci-a
+spec:
+  moduleLoader:
+    container:
+      modprobe:
+        moduleName: ooto-ci-a 
+
+      kernelMappings:
+        - literal: 5.4.196-108.356.amzn2.x86_64
+          containerImage: 804517629211.dkr.ecr.eu-west-3.amazonaws.com/ooto-kmod:2.0
+          build:
+            buildArgs:
+              - name: CI_BUILD_ARG
+                value: some-build-arg
+            pull:
+              insecure: false
+            push:
+              insecure: false
+            dockerfile: |
+              ARG KERNEL_VERSION=''
+              FROM amazonlinux as builder
+              RUN amazon-linux-extras install kernel-5.4 -y
+              RUN yum -y install kernel-headers-5.4.196-108.356.amzn2.x86_64 kernel-devel-5.4.196-108.356.amzn2.x86_64 kernel-5.4.196-108.356.amzn2.x86_64 python3 git make gcc bison flex elfutils-libelf-devel openssl-devel xz bc && yum clean all
+              RUN git clone https://github.com/enriquebelarte/ooto-kmod
+              WORKDIR ooto-kmod 
+              RUN make 
+    
+              FROM amazonlinux
+              RUN yum -y install kmod && yum clean all
+              ARG KERNEL_VERSION
+              RUN mkdir -p /opt/lib/modules/${KERNEL_VERSION}
+              COPY --from=builder /ooto-kmod/ooto_ci_a.ko /opt/lib/modules/${KERNEL_VERSION}/
+              COPY --from=builder /ooto-kmod/ooto_ci_b.ko /opt/lib/modules/${KERNEL_VERSION}/
+              RUN depmod -b /opt
```

Comment:
Same comments as before regarding the order of the commands and creating the dir.